tests/UI/pages/BO/quickAccess/index.js

```diff
@@ -19,6 +19,7 @@ class QuickAccess extends BOBasePage {
     // Selectors
     // Header selectors
     this.addNewQuickAccessButton = 'a[data-role=page-header-desc-quick_access-link]';
+    this.removeQuickAccessLink = '#quick-remove-link';
```

Comment:
Please delete the not used selector
Thanks!

Reply:
Done